### ISSUE TITLE
Replace the wheel by 3 buttons

### ISSUE THREAD

**Issue by Trim** [mooltipass/minible_hw#2]
Hello,

I don't know how far this new product is from shipping, but I wanted to do a hardware suggestion.

I'm owner of a Mooltipass Mini and it felt down by accident on the ground this night. It seems that the wheel took all the shock.

Now the Mooltipass Mini is broken because the wheel is pressed down and I've discovered, I won't be able to repair it [without breaking the case](https://fry.blueblue.fr/mpm/teardown/).

So, after this experience, I think the wheel is very fragile [1] and it should be replaced by three buttons to navigate on the menus (with the middle button as validating button).

The advantage of the button is they are more robust and, with software, we can simulate fast scroll when the button is kept pressed.

In conclusion, if this new Mini BLE can have bluetooth and the 3 buttons, I think the hardware will be way more robust, because the USB port will be less pluged in/out and the buttons will be well protected by the case.

[1]: even with normal usage, my wheel didn't work very well any more because some of the steps seemed to be to used and didn't give signal to the hardware.
